### PR TITLE
Webkul - ME-32

### PR DESCRIPTION
**Pull request by dhiraj-toptal**

### CODE REVIEW

**Comment by ynaxonBalance** [comment]
@dhiraj-toptal is this the terms feature for webkul module?

File: Setup/Patch/Data/TermOptions.php [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    /**
+     * {@inheritdoc}
+     */
+    public function apply()
+    {
+        $this->moduleDataSetup->getConnection()->startSetup();
+        /** @var CustomerSetup $customerSetup */
+        $customerSetup = $this->customerSetupFactory->create(['setup' => $this->moduleDataSetup]);
+        $customerSetup->removeAttribute(\Magento\Customer\Model\Customer::ENTITY, 'term_options');
```
Comment (ynaxonBalance) [inline Setup/Patch/Data/TermOptions.php:54]: import (use) in beginning of file

**Review by ynaxonBalance — APPROVED** [review]
Looking good, see comment before merging :)

**Comment by dhiraj-toptal** [comment]
Yes its a term feature for webkul module.
I am merging it, I will update above comment review in next PR.

**Pull request merged**